src/erf.jl

```diff
@@ -0,0 +1,143 @@
+
+ # see (http://git.musl-libc.org/cgit/musl/tree/src/math/erf.c) for implementation details
```

Comment:
Probably needs a copyright notice here: i.e. based on musl code, copyright whoever is on that code. Also would be good to add which git commit you used to make it easy to check for updates.